projects/frontend/shared-components/gui/version.txt

```diff
@@ -1 +1 @@
-0.0.0
+1.0.0-alpha.0
```

Comment:
same here; please use minor version in favour of patch

Reply:
Yes I understand the logic you explained to me before, and I'll change it according your guidance.

Reply:
Changed.